Rebase maintenance v0.14.2 to master candidate

tddang-linagora opened the pull request:
[0.14.12-firefox-safari.ods](https://github.com/user-attachments/files/18655643/0.14.12-firefox-safari.ods)

dab246 commented:
- [ ] Email reply still not working properly, such as case in demo video.



https://github.com/user-attachments/assets/734267a6-57d5-4d17-ba37-98955a8e7114


- [ ] IMO, We also should implement behavior email reply such as TB @hoangdat 


<img width="1203" alt="Screenshot 2025-02-05 at 10 09 31" src="https://github.com/user-attachments/assets/7e73b562-812d-4269-96c3-5931234e83a8" />

tddang-linagora commented:
>Email reply still not working properly, such as case in demo video.

Reverted back to old logic, ignored `replyTo` in "reply" action and added more test.

https://github.com/user-attachments/assets/48591804-fa82-48a3-a387-11472e28e290

hoangdat commented:
- [ ] when copy a list of email addresses to composer, it not auto convert to chips. Ref: https://github.com/linagora/tmail-flutter/issues/3344
  Fx: john.doe@example.com; jane.smith@example.com; alex.wilson@example.com
  
![image](https://github.com/user-attachments/assets/79011a90-2c92-4fd2-acf9-6c207cbac240)
